src/west/app/project.py

```diff
@@ -266,10 +275,13 @@ def bootstrap(self, args) -> Path:
         temp_manifest_filename = args.manifest_file or 'west.yml'
         temp_manifest = tempdir / temp_manifest_filename
         if not temp_manifest.is_file():
+            if args.manifest_rev:
+                and_rev = f' and --manifest-rev={args.manifest_rev}'
+            else:
+                and_rev = ''
             log.die(f'can\'t init: no {temp_manifest_filename} found in '
                     f'{tempdir}\n'
-                    f'  Hint: check --manifest-url={manifest_url} and '
-                    f'--manifest-rev={manifest_rev}\n'
+                    f'  Hint: check --manifest-url={manifest_url}{and_rev}\n'
```

Comment:
Nesting an f-string into another one makes  me miss the simplicity of concatenation with `+`